slam/src/main.rs

```diff
@@ -501,9 +500,13 @@ fn build_tx(
     let block_version = BlockVersion::try_from(BLOCK_VERSION.load(Ordering::SeqCst))
         .expect("Unsupported block version");
 
+    // Assume MOB token
+    let token_id = Mob::ID;
```

Comment:
Shouldn't it get it from the SpendableTxOuts? and panic if they are mixing multiple token ids?

Reply:
yeah that seems better